source/FreeRTOS_DHCP.c

```diff
@@ -100,7 +100,7 @@
 /*
  * Generate a DHCP request packet, and send it on the DHCP socket.
  */
-    static BaseType_t prvSendDHCPRequest( const NetworkEndPoint_t * pxEndPoint );
+    static BaseType_t prvSendDHCPRequest( NetworkEndPoint_t * pxEndPoint );
```

Comment:
Why are we removing const? They might have been added for MISRA fixes

Reply:
Thats becuase in the same function we are assigning it to a non const pointer (`xDHCPv4Socket->pxEndPoint = pxEndPoint;`) and IAR considers this as an error. I think MISRA shouldn't be complaining about that in this case.